c_glib/test/test-buffer-input-stream.rb

```diff
@@ -22,4 +22,20 @@ def test_read
     read_buffer = buffer_input_stream.read(5)
     assert_equal("Hello", read_buffer.data.to_s)
   end
+
+  def test_advance
+    buffer = Arrow::Buffer.new("Hello World")
+    buffer_input_stream = Arrow::BufferInputStream.new(buffer)
+    buffer_input_stream.advance(6)
+    read_buffer = buffer_input_stream.read(5)
+    assert_equal("World", read_buffer.data.to_s)
+  end
+
+  def test_align
+    buffer = Arrow::Buffer.new("Hello World")
+    buffer_input_stream = Arrow::BufferInputStream.new(buffer)
+    buffer_input_stream.advance(8)
```

Comment:
Perhaps should we test with `align ()` ?

Reply:
Oh! Good catch!
I've fixed this.